pkg/generators/models/testdata/cases/validation/expected/model_person.go

```diff
@@ -0,0 +1,218 @@
+// This file is auto-generated, DO NOT EDIT.
+//
+// Source:
+//     Title: Test
+//     Version: 0.1.0
+package generatortest
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-ozzo/ozzo-validation/v4/is"
+
+	"time"
+)
+
+// Person is an object.
+type Person struct {
+	// Age:
+	Age float64 `json:"age,omitempty"`
+	// Base64:
+	Base64 string `json:"base64,omitempty"`
+	// Date:
+	Date string `json:"date,omitempty"`
+	// Datetime:
+	Datetime time.Time `json:"datetime,omitempty"`
+	// Email:
+	Email string `json:"email,omitempty"`
+	// Gender:
+	Gender Gender `json:"gender"`
+	// Hostname:
+	Hostname string `json:"hostname,omitempty"`
+	// Ip:
+	Ip string `json:"ip,omitempty"`
+	// Ipv4:
+	Ipv4 string `json:"ipv4,omitempty"`
+	// Ipv6:
+	Ipv6 string `json:"ipv6,omitempty"`
+	// Name:
+	Name string `json:"name"`
+	// Url:
+	Url string `json:"url,omitempty"`
+	// Uuid:
+	Uuid string `json:"uuid,omitempty"`
+}
+
+// Validate implements basic validation for this model
+func (m Person) Validate() error {
+	return validation.Errors{
+		"age": validation.Validate(
+			m.Age, validation.Min(float64(18)), validation.Max(float64(120)),
```

Comment:
age is NOT required in the spec, so we allow age == 0 here

Reply:
that is an ugly edge case but i can't see a way around it while also matching the meaning of the openapi spec.  Seems like it is a limitation of Go and we would need a custom serializer to really fix it.

I wonder i the generator could produce a warning to stderr so that we have a chance to catch it before  a user does

Reply:
ok, so now `Age` is required because it has a minimum > 0 and is not nullable. I think now we got it :)